src/logtab.cpp

```diff
@@ -39,18 +39,20 @@ void LogTab::InitTreeView(const EventListPtr events)
 
     m_bar->ShowMessage(QString("%1 events loaded").arg(QString::number(m_treeModel->rowCount())), 3000);
 
+    bool isPlainText = (events->size() > 0 && events->at(0)["k"].toString().isEmpty());
```

Comment:
Our json-based log files all have keys in each line.  The purpose here is that if we can't parse or find a key in the first line/event, then there is no point showing the extra columns. This so far only happens with non-json plain text files like server and tde logs.
Perhaps I can change the var name to hasNoKey to be more technically correct.